app/services/funnel/registration/add_mfa.rb

```diff
@@ -3,14 +3,33 @@
 module Funnel
   module Registration
     class AddMfa
-      def self.call(user_id, mfa_method, analytics)
+      def self.call(user_id, mfa_method, analytics, threatmetrix_attrs = {})
         now = Time.zone.now
         funnel = RegistrationLog.create_or_find_by(user_id: user_id)
         return if funnel.registered_at.present?
 
         analytics.user_registration_user_fully_registered(mfa_method: mfa_method)
+        @device_profiling_result = process_threatmetrix_for_user(
```

Comment:
Is there a reason this should be assigned as an instance variable?

```suggestion
        device_profiling_result = process_threatmetrix_for_user(
```